dataglen_login/spiders/login.py

```diff
@@ -13,8 +13,8 @@ def parse(self, response):
         csrf_token = response.xpath('//*[@name = "csrfmiddlewaretoken"]/@value').extract_first()
         yield FormRequest('https://dataglen.com/api-auth/login/',
                           formdata = {'csrfmiddlewaretoken': csrf_token,
-                                      'username': 'solaripgcl@gmail.com',
```

Comment:
@assamidanov Environment variables come handy in cases like this. Check [this](https://www.geeksforgeeks.org/python-os-environ-object/) out